src/aai/aai.js

```diff
@@ -43,7 +43,7 @@
 
             // Encode the redirect URL
             targetUrl += window.encodeURIComponent(redirectUrl);
-            window.location = opts.host + '/Shibboleth.sso/Login?SAMLDS=1&target=' + targetUrl + '&entityID=' + window.encodeURIComponent(e.entityID);
+            window.location = opts.host + opts.port + '/Shibboleth.sso/Login?SAMLDS=1&target=' + targetUrl + '&entityID=' + window.encodeURIComponent(e.entityID);
```

Comment:
opts.port is in the format ":8843"? mind the `:`

Reply:
Yes it is, it worked for me locally